lib/puppet_x/certs/provider/keystore.rb

```diff
@@ -20,7 +20,7 @@ def exists?
               '-storepass:file', resource[:password_file],
             )
           rescue Puppet::ExecutionFailure => e
-            if e.message.include?('java.security.UnrecoverableKeyException')
+            if e.message.include?('java.security.UnrecoverableKeyException') || e.message.include?('keystore password was incorrect')
```

Comment:
I debated treating any error as "it doesn't exist" and still wonder if we should take that route.